pkgs/applications/editors/emacs/make-emacs.nix

```
@@ -78,21 +77,20 @@
   withGTK3 ? withPgtk && !noGui,
   withGlibNetworking ? withPgtk || withGTK3 || (withX && withXwidgets),
   withGpm ? stdenv.hostPlatform.isLinux,
-  withImageMagick ? lib.versionOlder version "27" && (withX || withNS),
-  # Emacs 30+ has native JSON support
-  withJansson ? lib.versionOlder version "30",
+  # https://github.com/emacs-mirror/emacs/blob/master/etc/NEWS.27#L140-L142
+  withImageMagick ? false,
   withMailutils ? true,
   withMotif ? false,
   withNS ? stdenv.hostPlatform.isDarwin && !(variant == "macport" || noGui),
   withPgtk ? false,
   withSelinux ? stdenv.hostPlatform.isLinux,
-  withSQLite3 ? lib.versionAtLeast version "29",
+  withSQLite3 ? lib.versionAtLeast version "30",
```

If the idea is to support only versions newer than 30, why not just default to true?
```suggestion
  withSQLite3 ? true,
```